urdf2webots/writeProto.py

```diff
@@ -389,6 +390,8 @@ def URDFVisual(proto, visualNode, level, normal=False):
                 for value in visualNode.geometry.trimesh.coordIndex:
                     if len(value) == 3:
                         proto.write('%d %d %d -1 ' % (value[0], value[1], value[2]))
+                    if len(value) == 2:
+                        proto.write('%d %d -1 ' % (value[0], value[1]))
             elif isinstance(visualNode.geometry.trimesh.coordIndex[0], np.int32):
                 for i in range(int(len(visualNode.geometry.trimesh.coordIndex) / 3)):
                     proto.write('%d %d %d -1 ' % (visualNode.geometry.trimesh.coordIndex[3 * i + 0],
```

Comment:
Normal and texcoord (just below) have to be disabled in case of lineset.

Reply:
A LineSet should not have those, right?

Reply:
Indeed lineset represents lines, there is not surface and therefore the normal and texture doesn't make sense (here is by the way the description of indexedLineSet: https://cyberbotics.com/doc/reference/indexedlineset).

Reply:
This part should be resolved right?

Reply:
Yes